js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedError.tsx

```diff
@@ -0,0 +1,24 @@
+import {Box, NonIdealState} from '@dagster-io/ui-components';
+
+import {ApolloError} from '../apollo-client';
+
+export const RunsFeedError = ({error}: {error: ApolloError | undefined}) => {
+  const badRequest = !!(
+    typeof error === 'object' &&
+    'statusCode' in error &&
+    error.statusCode === 400
+  );
```

Comment:
I remember why we added this. I wonder if we should be doing this in more places, now that we're adding filtering on so many of our list views. (No action required here, just making a note of it.)

Reply:
Hmm interesting - maybe so... It feels sort of odd that this isn't a GraphQL error and that invalid filters is an HTTP-level error?